src/main.py

```diff
@@ -37,7 +38,10 @@ async def main(jobs: list[Job]) -> None:
     """
     tasks = [job.run() for job in jobs]
     for completed_task in asyncio.as_completed(tasks):
-        await completed_task
+        try:
+            await completed_task
+        except Exception as e:  # pylint: disable=broad-exception-caught
+            log.critical("Error in job execution: %s", str(e))
```

Comment:
Not entirely sure if this is the best idea... we are sort of swallowing any exception here as a critical log. Make a bit of sense because we wouldn't want the entire program to crash if only one job crashes... what do you think @mooster531 ?

Reply:
Agreed, CRITICAL is too high, probably best to make it ERROR.